src/components/CallView/shared/VideoBackground.vue

```diff
@@ -64,6 +89,22 @@ export default {
 	top: 0;
 	height: 100%;
 	width: 100%;
+	&__picture {
+		filter: blur(20px);
```

Comment:
@Ivansss what is the blur intensity you're using on iOS?
I remember checking this last year at Stuttgart with you & Jan :)

Reply:
On iOS you need to set the inputRadius of the filer.
We finally set it to:
`CGFloat inputRadius = 8.0f;`
But I don't know how to convert that into blur pixels :S

Reply:
I use 5.0f. Should I set it to 8?

Reply:
I think we should be consistent yes :)

Reply:
> But I don't know how to convert that into blur pixels :S

Hum, good question. Both are gaussian blur. Is this a percentage for you @Ivansss, @mario ?

Reply:
For css: "blur: Applies a Gaussian blur to the input image. The value of ‘radius’ defines the value of the standard deviation to the Gaussian function"

Reply:
https://developer.apple.com/documentation/coreimage/cigaussianblur?language=objc

According to the docs, the radius is also in px, but you're passing it as float, so we should use 8px as well I assume :)

Reply:
Can we compare on the same image so we make sure this is the same? Can you post a screenshot @Ivansss ?

Reply:
8px is way too low here. We have overlay text and controls and the beckground could become very distracting. I don't see the need of matching the mobile radius here

Reply:
This is how it looks on iOS:
![IMG_0281](https://user-images.githubusercontent.com/4638605/80363313-574dd280-8884-11ea-8de4-019128cff89a.PNG)

Reply:
This has an `inputGradient = 8.0` and a black top layer with `alpha = 0.6`

Reply:
> 8px is way too low here

Agreed @ma12-co, I guess this doesn't translate properly.
Can you tweak things a bit so you're reaching the similar design for iOS ?
Maybe it's factor 10 ? 80px? Only god knows! :grin: